extensions/README.md

```diff
@@ -79,6 +104,20 @@ with existing extensions as well as possible and may even re-use fields and thei
 into a new extension that combines commonly used fields across multiple extensions.
 Best practices for extension proposals are still emerging in this section.
 
+### General Conventions
+
+Creating a new extension involves defining a set of logically grouped fields, and specifying what the allowed values
```

Comment:
Not all extensions may define fields, some may just define behavior (e.g. formerly the Commons extension) or define e.g. just rel types or asset roles.